README.md

```diff
@@ -64,12 +64,12 @@ Evidently, there is great potential for the future of healthcare with smart tool
 ### Potential Benefits Of Deep Learning in Healthcare?
 Among the several benefits artificial intelligence promises to bring to healthcare, these are worth highlighting.
   * **More affordable treatment:** Diagnosis is faster and more accurate with automation; doctors will be able 
-  to recommend the right treatments to patients or intervene appropriately before illnesses aggravate to 
+  to recommend the right treatments to patients or intervene appropriately before illnesses aggravates which
```

Comment:
to recommend the right treatments to patients or intervene appropriately before illnesses aggravate leading to more expensive treatment options.

Reply:
to recommend the right treatments to patients or intervene appropriately before illnesses aggravate, leading to more expensive treatment options.